yarn-project/pxe/src/pxe_service/pxe_service.ts

```diff
@@ -877,7 +897,7 @@ export class PXEService implements PXE {
         }
 
         let validationTime: number | undefined;
-        if (!skipTxValidation) {
+        if (!skipTxValidation && !skipKernels) {
```

Comment:
Would it ever be correct that `skipKernels` would be true and `skipTxValidation` would be false? Feels like if `skipKernels` is true then `skipTxValidation` should alway be true, no? Would it make sense to error out if we get `skipKernels` true and `skipTxValidation` false?